gpkit/nomials.py

```diff
@@ -492,22 +490,14 @@ def __pow__(self, other):
 
 
 class Constraint(Posynomial):
-    """A constraint of the general form monomial > posynomial
-    Stored internally (exps, cs) as a single Posynomial (1 >= self)
+    """A constraint of the general form posynomial <= monomial
```

Comment:
I generally find monomial >= posynomial to be easier to read, just because it has a smaller left-hand side...